scripts/mac/generic/zluri-generic-mac-hide-icon-script.sh

```
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# Get current user and home directory
+CURRENT_USER=$(echo "show State:/Users/ConsoleUser" | scutil | awk '/Name :/ && ! /loginwindow/ { print $3 }')
+HOMEDIR=$(/usr/bin/dscl . -read /Users/"$CURRENT_USER" NFSHomeDirectory | /usr/bin/cut -d' ' -f2)
+
+# Configuration variables
+ORG_TOKEN=<orgToken> # needs to be added by the customer
```

Shellcheck (0.11.0)

[error] Couldn't parse this variable assignment. Fix to allow more checks.

(SC1073)